.buildkite/test.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -euo pipefail
+
+python ./test.py
+
+# shard_targets () {
```

Comment:
Is the commented out code still here on purpose?

Reply:
This is a Draft pull request so WIP  – you don't have to review them ;)